File: executions/cpp/execution-a00458fe-79b3-49dc-a88e-5373b6f12597/sourceCode.cpp
#include <iostream>
#include <cstdlib>
#include <string>
using namespace std;

int calculatePower(int, int);

int main(int argc, char* argv[])
{
    int base, powerRaised, result;

    int a = std::stoi(argv[1]);
    int b = std::stoi(argv[2]);
    result = calculatePower(a, b);
    std::cout  << result;

    return 0;
}

int calculatePower(int base, int powerRaised)
{
    if (powerRaised != 0)
        return (base*calculatePower(base, powerRaised-1));
    else
        return 1;
}

File: executions/python/execution-8c4a555f-9fd0-430d-85e7-67c4b3b87478/sourceCode.py
import sys

def power(base, exp):
    if (exp == 1):
        return (base)
    if (exp != 1):
        return (base * power(base, exp - 1))
base = int(sys.argv[1])
exp = int(sys.argv[2])
print(power(base, exp))
